Libs/MRML/Core/vtkMRMLDoubleArrayNode.h

```diff
@@ -100,7 +100,16 @@ class VTK_MRML_EXPORT vtkMRMLDoubleArrayNode : public vtkMRMLStorableNode
   /// int GetXYAxisValue(int index, double* x, double* y, double*  yerr);
 
   ///
-  /// Set X and Y values at he data point specified by 'index'
+  /// Set values at the data point specified by 'index'
+  int SetValues(int index, double* values);
+
+  ///
+  /// Set value of the component specified by 'component'
+  /// at the data point specified by 'index'
```

Comment:
Wouldn't this be more adequate :
```
/// Set value for the specified data point 'index' at the given 'component'
```
or to be consistent with other comments:
```
/// Set value at the data point specified by 'index' at the given 'component'
```
?